docs/building/environment.md

```diff
@@ -0,0 +1,153 @@
+# Creating a local building environment
+
+> These instructions are meant to describe how to create a local building and
+> execution environment. If you would like to build tracee container(s)
+> image(s), [read this](./containers.md) instead.
+
+## Quick steps (**impatient readers**)
+
+* Build and execute `tracee-ebpf`:
+
+  ```
+  $ make -f builder/Makefile.tracee-make alpine-prepare
+  $ make -f builder/Makefile.tracee-make alpine-shell
+
+  tracee@f64bb4a2f0b1[/tracee]$ make -f Makefile.one clean
+  tracee@f64bb4a2f0b1[/tracee]$ make -f Makefile.one tracee-ebpf
+  tracee@f64bb4a2f0b1[/tracee]$ sudo ./dist/tracee-ebpf \
```

Comment:
These instructions are for impatient readers who have BTF-enabled kernels, i.e. they won't run `make -f Makefile.one install-bpf-nocore`. Is that correct?

Reply:
Yes, I assumed most of the readers interested in building `tracee-ebpf` would either be aware of that (after reading docs, at least) OR face an error and investigate (since `install-bpf-nocore` needs some understanding of what is going on.